src/screens/OverviewScreen.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import { Icon } from 'react-native-elements'
+
+const OverviewScreen = props => {
+  return (
+
+    <View style={styles.container}>
+      <View style={styles.body}>
+
+        <Text>OverviewScreen (/Map)</Text>
+
+      </View>
+
+      <View style={styles.navigationButtons}>
+        <TouchableOpacity onPress={() => {
+          props.navigation.navigate({ routeName: 'Home' })
+        }}
+        >
+          <Icon style={styles.iconNavigation} name='arrow-left-circle' type='feather' size={35} />
+        </TouchableOpacity>
+
+        <TouchableOpacity onPress={() => {
+          props.navigation.navigate({ routeName: 'Task' })
+        }}
+        >
+          <Icon style={styles.iconNavigation} name='arrow-right-circle' type='feather' size={35} />
+        </TouchableOpacity>
+      </View>
+
+    </View>
+  )
+}
+
+OverviewScreen.navigationOptions = (navData) => {
+  return {
+    headerShown: false
+  }
+}
+
+const styles = StyleSheet.create({
```

Comment:
Question: Do we need a stylesheet for each page or can we reuse common definitions for all pages and merge them with specific styles on each page? No need to implement this, I am just seeing a pattern here. Let's talk about it soon.